### ISSUE TITLE
Spectrum speed control (DeathAdder V2)

### ISSUE THREAD

**Issue by kneekna** [1kc/razer-macos#315]
Would it be possible to change the speed at which the spectrum setting changes colors?

**Comment by dhobi** [comment]
Yes and no.

No: 
The built in hardware spectrum command does not take a speed parameter (if I'm not mistaken).

Yes:
One workaround is to fake the spectrum effect by changing the color in a fixed interval. This is already done in Razer macOS (All devices -> Spectrum -> Animation). 

What we would need to do is to:
- provide a spectrum animation for each device. Currently, you can only activate the hardware spectrum. 
- allow to specify the interval in seconds

Does this make sense?

**Comment by kneekna** [comment]
Yes, that makes sense! So currently there is no option to change the interval of color change? Are there plans to implement this feature in the future?

Great work so far though! I’m certainly no expert when it comes to any of this so I applaud all your efforts and progress so far :)
